app/controllers/finance/ecf/payment_breakdowns_controller.rb

```diff
@@ -11,14 +11,64 @@ def show
           contract: @ecf_lead_provider.call_off_contract,
           event_type: :started,
         )
-        @cutoff_date = "On #{helpers.cutoff_date}"
+
+        @payment_date = current_payment_date
+        @deadline_date = current_deadline_date
+      end
+
+      def payable
+        @ecf_lead_provider = lead_provider_scope.find(params[:id])
+
+        @breakdown = CalculationOrchestrator.call(
+          aggregator: ParticipantPayableAggregator,
+          cpd_lead_provider: @ecf_lead_provider.cpd_lead_provider,
+          contract: @ecf_lead_provider.call_off_contract,
+          event_type: :started,
+        )
+
+        @payment_date = payable_payment_date
+        @deadline_date = payable_deadline_date
+
+        render :show
       end
 
     private
 
       def lead_provider_scope
         policy_scope(LeadProvider, policy_scope_class: FinanceProfilePolicy::Scope)
       end
+
+      def payable_milestone
```

Comment:
so there does not seem to be a concept of "bucketing declaration to an invoice" or at least yet. So as far as I can tell when something is paid we simply know it is in the paid state but not which invoice it belongs to.

This is why these milestones and the dates are separate (for) now and hardcoded. till these concepts are worked in and we can query these in a correct manner

Reply:
I don't see invoices as being within the current scope of the service right now. In some ways we don't need to know. We'd just expect to just be told when something has been paid by finance.